contrib/babelfishpg_tds/src/backend/tds/tdstypeio.c

```diff
@@ -1963,6 +1979,49 @@ TdsRecvTypeDatetime2(const char *message, const ParameterToken token)
 	return result;
 }
 
+/* -------------------------------
+ * TdsRecvTypeGeometry - converts external binary format to
+ * Geometry data type
+ * --------------------------------
+ */
+Datum
+TdsRecvTypeGeometry(const char *message, const ParameterToken token)
+{
+	Datum result = 0; 
+
+	//Decode binary and convert if needed 
+	StringInfo	buf = TdsGetStringInfoBufferFromToken(message, token);
+
+	//Return in Datum val
+
+	ereport(ERROR,
+							(errcode(ERRCODE_DATETIME_VALUE_OUT_OF_RANGE),
+							 errmsg("tdstypeio.c: TdsRecvTypeGeometry")));    
+
+	pfree(buf); 
+	return result; 
+}
+
+/* -------------------------------
+ * TdsRecvTypeGeography - converts external binary format to
+ * Geography data type 
+ * --------------------------------
+ */ 
+Datum
+TdsRecvTypeGeography(const char *message, const ParameterToken token)
```

Comment:
Same comments as above in `TdsRecvTypeGeometry` also apply here.

Reply:
Addressed